src/rollup/RollupCreator.sol

```diff
@@ -102,29 +102,35 @@ contract RollupCreator is Ownable {
      *                          anyone can try to deploy factories and potentially burn the nonce 0 (ie. due to gas price spike when doing direct
      *                          L2 TX). That would mean we permanently lost capability to deploy deterministic factory at expected address.
      *          - maxFeePerGasForRetryables price bid for L2 execution.
+     *          - dataHashReader The address of the data hash reader used to read blob hashes
      * @return The address of the newly created rollup
      */
     function createRollup(RollupDeploymentParams memory deployParams)
         public
         payable
         returns (address)
     {
-        // Make sure the immutable maxDataSize is as expected
-        (, ISequencerInbox ethSequencerInbox, IInboxBase ethInbox, , ) = bridgeCreator
-            .ethBasedTemplates();
-        require(
-            deployParams.maxDataSize == ethSequencerInbox.maxDataSize(),
-            "SI_MAX_DATA_SIZE_MISMATCH"
-        );
-        require(deployParams.maxDataSize == ethInbox.maxDataSize(), "I_MAX_DATA_SIZE_MISMATCH");
+        {
```

Comment:
Why this change?

Reply:
Stack size with the new `dataHashReader ` in `RollupDeploymentParams `

Reply:
Weird, I would have expected there just to be a mem pointer to that struct on the stack, not all it's contents. That datahashreader property isnt used in this function so why is it on the stack?